pulseapi/settings.py

```diff
@@ -31,6 +31,8 @@
     CORS_ORIGIN_REGEX_WHITELIST=(list, []),
     CORS_ORIGIN_WHITELIST=(list, []),
     CSRF_TRUSTED_ORIGINS=(list, []),
+    CRM_AWS_SQS_ACCESS_KEY_ID=(str, None),
```

Comment:
Do these need to go in the `sample.env`?

Reply:
shouldn't be necessary, since environ is setting None when they're not defined in the environment.

Reply:
on second thought, might be helpful to have them there, as empty strings with a quick description or something?

Reply:
added